components/AutoComplete/index.d.ts

```diff
@@ -12,9 +12,10 @@ export interface AutoCompleteProps {
   helperText?: string;
   error?: string;
   disabled?: boolean;
-  placeholder?: string;
-  onChange?: () => void;
-  onSelectedItem?: () => void;
+  minlength?: string;
+  onChange?: (item: string) => void;
+  selectedItem?: (item: string) => void;
```

Comment:
There isn't this prop on component

Reply:
I'm sorry man, I didn't notice `minLength` prop, but `selectedItem` isn't on component.